src/Sulu/Bundle/AdminBundle/Resources/js/components/ItemSelection/Header.js

```diff
@@ -0,0 +1,48 @@
+// @flow
+import React from 'react';
+import classNames from 'classnames';
+import type {Button as ButtonConfig} from './types';
+import Button from './Button';
+import headerStyles from './header.scss';
+
+type Props = {
+    label?: string,
+    emptyList: boolean,
```

Comment:
Is `emptyList` a good name? I think when we are talking about a `ItemSelection` with the possibility of adding multiplte items, the header should always look the same, shouldn't it?